plugins/modules/inventory_workflow_manager.py

```diff
@@ -3836,6 +4142,89 @@ def verify_diff_deleted(self, config):
 
         return self
 
+    def update_inventory_profile_messages(self):
+        """
+        Updates and logs messages based on the status of users and roles.
+        Args:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+        Returns:
+            self (object): Returns the current instance of the class with updated `result` and `msg` attributes.
+        Description:
+            This method aggregates status messages related to the creation, update, or deletion of users and roles.
+            It checks various instance variables (`create_user`, `update_user`, `no_update_user`, `delete_user`,
+            `create_role`, `update_role`, `no_update_role`, `delete_role`) to determine the status and generates
+            corresponding messages. The method also updates the `result["response"]` attribute with the concatenated status messages.
+        """
+
+        self.result["changed"] = False
+        result_msg_list = []
```

Comment:
existing_device_messages instead of result_msg_list
 would hold messages about devices that are already present or cannot be deleted.

Reply:
it holds for adding devices deleting devices and provision of the devices

Reply:
Okay, then you can give a proper name? 
result_msg_list and result_msg_list_1. 

_1, _2, ... are not a way to name variables...

Reply:
Updated the code